test/rest-api/response-patching.test.ts

```diff
@@ -51,4 +51,42 @@ describe('REST: Response patching', () => {
       })
     })
   })
+
+  describe('given a post request to be patched', () => {
+    it('should be able to properly request and patch a post', async () => {
+      const REQUEST_URL = 'https://jsonplaceholder.typicode.com/posts'
+
+      const data = {
+        url: REQUEST_URL,
```

Comment:
There's no need to provide the `url` property to the `RequestInit` object if you supply it as the first argument to `fetch()` (which you do):

```js
api.page.evaluate((url, init) => fetch(url, init), REQUEST_URL, data)
```

Reply:
done.